database/mssql/test/versions/v_6_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Retrieve arguments
+source ${SCRIPT_DIR}/utility/getopt.sh
+USAGE="-u USER -p PASS -s SERVER -d DATABASE"
+parse_options "${USAGE}" ${@}
+
+# All database tests for database version 5 are run from this shell script.
+# TESTS_DIR variable set by the calling test-runner script.
+
+# Test 5.1 - verify that the pdf template table exists
+TEST_5_1_RESULT=$(/opt/mssql-tools/bin/sqlcmd -U ${USER} -P "${PASS}" -S ${SERVER} -v DB_NAME=${DATABASE} -h -1 -i ${TESTS_DIR}/v_6_1_test.sql)
```

Comment:
Should be TEST_6_1_RESULT (and below as well)

Reply:
Resolved